packages/perseus-score/src/util/answer-types.ts

```diff
@@ -230,13 +235,20 @@ const KhanAnswerTypes = {
 
                 // A proper fraction
                 proper: function (text) {
-                    return $.map(fractionTransformer(text), function (o) {
-                        // All fractions that are less than 1
-                        if (Math.abs(o.value) < 1) {
-                            return [o];
-                        }
-                        return [];
-                    });
+                    const transformed = fractionTransformer(text);
+                    return transformed.reduce(
```

Comment:
jQuery's `map` seemingly does some special flattening/filtering that's not as straight-forward as JS' `map`: https://github.com/HubSpot/youmightnotneedjquery/issues/355

Reply:
Agreed.  jQuery's `map` says for the mapping callback:

```
The function can return:

* the translated value, which will be mapped to the resulting array
* null or undefined, to remove the item
* an array of values, which will be flattened into the full array
```

That last point comes into play.  I think you could just use [`flatMap()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) here instead of `reduce()`

<img width="397" alt="image" src="https://github.com/user-attachments/assets/f1930713-78d8-46a2-bb76-61ba9c01395f" />

Reply:
Good call out, thank you